### PR TITLE
Requests and events controllers

### PR DESCRIPTION
**Pull request by msanchezb**
Some new views so that we can test the app

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -1,22 +1,66 @@
...
+    @event.user = current_user
+    if @event.save
+      redirect_to root_path
+    else
+      render :new
+    end
   end
 
   def show
+    @event = Event.find(params[:id])
```
Comment (freddiescadding) [inline app/controllers/events_controller.rb]: Remove this line

File: app/controllers/events_controller.rb [hunk 2]
```diff
@@ -1,22 +1,66 @@
...
   end
 
   def edit
+    @event = Event.find(params[:id])
```
Comment (freddiescadding) [inline app/controllers/events_controller.rb]: Remove this line

File: app/controllers/events_controller.rb [hunk 3]
```diff
@@ -1,22 +1,66 @@
...
   def update
+    if @event.update(event_params)
+      redirect_to event_path(@event)
+    else
+      render :edit
+    end
   end
 
   def destroy
+    @event.destroy
```
Comment (freddiescadding) [inline app/controllers/events_controller.rb]: Do we want a redirect?

File: app/controllers/events_controller.rb [hunk 4]
```diff
@@ -1,22 +1,66 @@
...
+  end
+
+  private
+
+  def event_params
+    params.require(:event).permit(:date, :location, :description, :category, :group_size, :latitude, :longitude)
```
Comment (freddiescadding) [inline app/controllers/events_controller.rb]: remove lat + long

File: app/controllers/requests_controller.rb [hunk 5]
```diff
@@ -1,8 +1,30 @@
...
+    @request.event = Event.find(params[:event_id])
+    if @request.save
+      redirect_to user_path(current_user)
+    else
+      render "events/#{params[:event_id]}"
+    end
   end
 
   def destroy
+    @request.destroy
```
Comment (freddiescadding) [inline app/controllers/requests_controller.rb]: redirect?

File: app/views/events/_form.html.erb [hunk 6]
```diff
@@ -0,0 +1,11 @@
+
+<%= simple_form_for @event do |f| %>
+  <%= f.input :description %>
+  <%= f.input :location %>
+  <%= f.input :date %>
+  <%= f.input :category %>
+  <%= f.input :group_size %>
+  <%= f.input :latitude %>
```
Comment (freddiescadding) [inline app/views/events/_form.html.erb]: remove lines 8 + 9

File: app/views/shared/_navbar.html.erb [hunk 7]
```diff
@@ -39,6 +39,9 @@
       </div>
     <% else %>
       <!-- Login link (when logged out) -->
+      <% unless current_user.nil?  %>
```
Comment (freddiescadding) [inline app/views/shared/_navbar.html.erb]: if user_signed_in? may be cleaner

File: app/controllers/requests_controller.rb [hunk 8]
```diff
@@ -1,8 +1,30 @@
...
   def create
+    @request.new(request_params)
+    @request.user = current_user
+    @request.event = Event.find(params[:event_id])
+    if @request.save
+      redirect_to user_path(current_user)
+    else
+      render "events/#{params[:event_id]}"
+    end
   end
```
Comment (freddiescadding) [inline app/controllers/requests_controller.rb]: Do we want an update method?

Comment (msanchezb) [reply app/controllers/requests_controller.rb]: The update method is not necessary, but we need a bunch of others, to confirm the event and modify it, so it probably needs to exist

File: app/controllers/events_controller.rb (hunk 3 above)
Comment (msanchezb) [reply app/controllers/events_controller.rb]: Done. To root for the moment since it works fine

File: app/controllers/events_controller.rb (hunk 1 above)
Comment (msanchezb) [reply app/controllers/events_controller.rb]: Done

File: app/controllers/events_controller.rb (hunk 2 above)
Comment (msanchezb) [reply app/controllers/events_controller.rb]: Done

File: app/controllers/events_controller.rb (hunk 4 above)
Comment (msanchezb) [reply app/controllers/events_controller.rb]: Done

Comment (msanchezb) [reply app/controllers/events_controller.rb]: Actually, if we intend to fetch the latitude and longitude through AJAX with js we might need it

File: app/views/shared/_navbar.html.erb (hunk 7 above)
Comment (msanchezb) [reply app/views/shared/_navbar.html.erb]: Done

File: app/controllers/events_controller.rb (hunk 4 above)
Comment (msanchezb) [reply app/controllers/events_controller.rb]: Let's go with it for the moment. We can add them later on if we need it

**Pull request closed without merging**